controllers/playlists.ts

```diff
@@ -140,7 +143,39 @@ export const getPlaylist = async (req, res, next) => {
     .where("playlist_track.playlist_id", id)
     .orderBy("order", "asc");
 
-  res.json({ success: true, data: trackInfo });
+  if (sort === SORT_BY_SATS) {
+    const BEGIN_DATE = new Date("2023-10-01");
+    const END_DATE = new Date("2024-4-20");
```

Comment:
To make sure we're filtering at the right time relative to UTC, we might need to be more specific. Something like this?
```suggestion
    const BEGIN_DATE = new Date(Date.UTC(2023, 10, 1, 0, 0, 0)) 
    const END_DATE = new Date(Date.UTC(2023, 4, 20, 0, 0, 0)) 
```

Reply:
Tested locally, result is the same either way. We can leave as-is